x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rule_status_dropdown.tsx

```diff
@@ -86,12 +87,12 @@ export const RuleStatusDropdown: React.FunctionComponent<ComponentOpts> = ({
       ...(message.details && {
         text: toMountPoint(
           <ToastWithCircuitBreakerContent>{message.details}</ToastWithCircuitBreakerContent>,
-          { i18n: i18nStart, theme }
+          { i18n: i18nStart, theme, userProfile }
```

Comment:
Not passing a large object here for the services, because this is within a `useCallback`. See https://github.com/elastic/kibana/pull/202606#discussion_r1871588439